commands/bug.go

```diff
@@ -17,8 +19,16 @@ func Version() string {
 	return resp
 }
 
-func GetBug(id string) Defects {
-	url := fmt.Sprintf("bug/%v", id)
+// GetBug : Fetch a bug/s matching an id or a set of keywords. Also supports adding a limit of results.
+func GetBug(filter, limit string) Defects {
```

Comment:
Is `limit` supposed to represent how many results to return? If so, how do you feel about naming it `count` or `quantity`? Even though it won't match the API attribute name, it makes more sense in my opinion.

Reply:
Yes, perhaps. `count` seems good or maybe `resultCount`. Although I wouldn't change the flag name to `count` or something else because `limit` is more intuitive.